install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -131,30 +134,38 @@ spec:
                   custom:
                     properties:
                       url:
+                        description: URL of the custom Broker to connect to
                         type: string
                     type: object
                   native:
                     properties:
                       name:
+                        description: Name of the native Broker to connect to
```

Comment:
It'd be great to get more on the intended use of this name. Does this name get exposed in Meshery UI?

Reply:
The intended name here is name of the Broker instance to be deployed.